mcodingbot/plugins/python.py

```diff
@@ -0,0 +1,47 @@
+import re
+
+import crescent
+import hikari
+
+from mcodingbot.utils import Plugin
+
+plugin = Plugin()
+
+PEP_REGEX = re.compile(r"pep *(?P<pep>\d{1,4})", re.IGNORECASE)
+
+
+def get_pep_link(pep_number: int) -> str:
+    return f"<https://peps.python.org/pep-{pep_number:04}/>"
+
+
+@plugin.include
+@crescent.command(
+    name="pep", description="Find a Python Enhancement Proposal."
+)
+class PEPCommand:
+    pep_number = crescent.option(int, "the PEP number")
```

Comment:
```suggestion
    pep_number = crescent.option(
        int, 
        "The PEP number.", 
        name="pep-number", 
        min_value=0, 
        max_value=9999, 
    )
```

(suggestion so it's easier for you)